src/modules/plymouthcfg/plymouthcfg.conf

```diff
@@ -1,4 +1,38 @@
 ---
-# The plymouth theme to be set if plymouth binary is present
-# leave commented if packaged default theme should be used
+# Plymouth Configuration Module
+
```

Comment:
In a long comment / documentation block, please also give lines between paragraphs of the documentation a leading `#`; use a blank line to separate sections of documentation related to different configuration options. Here, everything discusses either the module as a whole, or the single configuration option, so every line should start with `#` (it's one section).